api/app/routes/chat.py

```
@@ -116,10 +136,17 @@ async def query(
             response_time=result["response_time"],
         )
 
+        # Record metrics to Prometheus
+        total_time = time.time() - start_time
+        QUERY_TOTAL.inc()
+        QUERY_RESPONSE_TIME_HISTOGRAM.observe(total_time)
+        CURRENT_RESPONSE_TIME.set(total_time)
+
         return JSONResponse(content=response_data.model_dump())
     except Exception as e:
         logger.error(f"Unexpected error processing request: {str(e)}")
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)